src/main/java/com/hibernate/model/MovieSession.java

```diff
@@ -0,0 +1,53 @@
+package com.hibernate.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private Movie movie;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
```

Comment:
let's use `@JoinColumn` annotation so that our columns would have the `right_naming` - and not `camelCaseName`